docs/changes/12006.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Updated the bundled CFITSIO library to 4.0.0. When compiling with an external
```

Comment:
Is this technically a bug? 🤔

Reply:
Not really but we have been doing that in the past I think. It would be useful for packagers to be able to package astropy 4.3 with cfitsio 4.0

Reply:
Sure, but should it be under "others" section instead of "bug fix"?

Reply:
Oups, I meant to write "other", I guess we really need the pre-commit check ;)

Reply:
If people start updating their system cfitsio independently of astropy, it might become a bug...